sklearn/utils/estimator_checks.py

```diff
@@ -77,8 +77,8 @@
                 'RANSACRegressor', 'RadiusNeighborsRegressor',
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
 
-ALLOW_NAN = ['Imputer', 'SimpleImputer', 'MICEImputer',
-             'MinMaxScaler', 'StandardScaler', 'QuantileTransformer']
+ALLOW_NAN = ['Imputer', 'SimpleImputer', 'ChainedImputer',
```

Comment:
Add the `StandardScaler` back here to remove the error.

Reply:
Oh dang that was my bad on the merging. Thanks!